TrafficCapture/captureProtobufs/build.gradle

```diff
@@ -9,11 +9,11 @@ plugins {
 
 dependencies {
     api project(":commonDependencyVersionConstraints")
-    api group: 'com.google.protobuf', name: 'protobuf-java', version: '3.22.2'
+    api group: 'com.google.protobuf', name: 'protobuf-java', version: '3.25.5'
```

Comment:
Can we remove the version from here?

Reply:
Let me test, I am also curious about this